packages/plugins/Zoom/src/store/index.ts

```diff
@@ -50,6 +51,15 @@ export const makeStoreModule = () => {
       decreaseZoomLevel({ dispatch, getters: { zoomLevel } }): void {
         dispatch('setZoomLevel', zoomLevel - 1)
       },
+      smoothZoom({ rootGetters: { map }, getters: { zoomLevel } }): void {
+        const view = map.getView()
+        if (view) {
+          view.animate({
+            zoom: zoomLevel,
+            duration: 1000,
```

Comment:
```suggestion
            duration: 500,
```
A second felt a bit too slow